### PR TITLE
Adds a CentCom dogborg

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
This PR adds CENTCOM style borgs, as well as a lawset for it and a module picker that lets a borg become a CENTCOM borg.
This PR is fully tested, and all features should work as intended. If not, please mention below.

## Why It's Good For The Game
image

## Changelog

:cl:
add: CENTCOM borg module and associated sprites
Add: CENTCOM lawset and law board
Add: A CENTCOM borg module, a paper apparatus that can grip paper.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/altborgs/code/robot_model.dm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		/obj/item/borg/paperplane_crossbow
+	)
+	special_light_key = null
+	borg_skins = list(
+		"Vale" = list(SKIN_ICON_STATE = "valecc", SKIN_ICON = CYBORG_ICON_CENTCOM_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE))
+	)
+
+/obj/item/robot_model/centcom/rebuild_modules()
+	..()
+	var/mob/living/silicon/robot/cyborg = loc
```
Comment (nevimer) [inline modular_zubbers/code/modules/altborgs/code/robot_model.dm:34]: loc is unreliable. consider using src if it works.

File: modular_zubbers/code/modules/altborgs/code/robot_upgrade.dm [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/obj/item/borg/upgrade/transform/ntjack
+	name = "borg module picker (Centcom)"
+	desc = "Allows you to to turn a cyborg into a experimental nanotrasen cyborg."
+	icon_state = "cyborg_upgrade3"
+	new_model = /obj/item/robot_model/centcom
+
+/obj/item/borg/upgrade/transform/ntjack/action(mob/living/silicon/robot/cyborg, user = usr)
+	return ..()
```
Comment (nevimer) [inline modular_zubbers/code/modules/altborgs/code/robot_upgrade.dm:8]: i dont think there is a point to even having an override here

File: modular_zubbers/code/modules/mob/living/silicon/robot/robot_defines.dm [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/mob/living/silicon/robot/model/centcom
+	icon = 'modular_zubbers/icons/mob/silicon/widerobot_cc.dmi'
+	icon_state = "valecc"
+	faction = list(ROLE_DEATHSQUAD)
+	req_access = list(ACCESS_CENT_GENERAL)
+	lawupdate = FALSE
+	scrambledcodes = TRUE // These are not station borgs.
+	ionpulse = TRUE
+	var/playstyle_string = "<span class='big bold'>You are a Central Command cyborg!</span><br>"
```
Comment (nevimer) [inline modular_zubbers/code/modules/mob/living/silicon/robot/robot_defines.dm:9]: there is a macro for spans

File: modular_zubbers/code/modules/mob/living/silicon/robot/robot_defines.dm [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	set_model = /obj/item/robot_model/centcom
+	cell = /obj/item/stock_parts/cell/bluespace
+
+/mob/living/silicon/robot/model/centcom/Initialize(mapload)
+	laws = new /datum/ai_laws/central_override()
+	laws.associate(src)
+	. = ..()
+	addtimer(CALLBACK(src, PROC_REF(show_playstyle)), 0.5 SECONDS)
```
Comment (nevimer) [inline modular_zubbers/code/modules/mob/living/silicon/robot/robot_defines.dm:17]: why? why not invoke_async or something

File: modular_zubbers/code/modules/mob/living/silicon/robot/robot_defines.dm (hunk 3 above)
Comment (ReturnToZender) [reply modular_zubbers/code/modules/mob/living/silicon/robot/robot_defines.dm:9]: I just took the original code, what would you suggest I place here instead?

**Pull request merged**